### ISSUE TITLE
Box 'proycon/lamachine' could not be found.

### ISSUE THREAD

**Issue by nikoltoll** [proycon/LaMachine#93]
Using the standard instructions on installing laMachine using vagrant/virtualbox fails. 

Console output
`> vagrant init proycon/lamachine
A 'Vagrantfile' has been placed in this directory. You are now
ready to 'vagrant up' your first virtual environment! Please read
the comments in the Vagrantfile as well as documentation on
'vagrantup.com' for more information on using Vagrant.
PS D:\lamachine> vagrant up
Bringing machine 'default' up with 'hyperv' provider...
==> default: Verifying Hyper-V is enabled...
==> default: Box 'proycon/lamachine' could not be found. Attempting to find and install...
    default: Box Provider: hyperv
    default: Box Version: >= 0
==> default: Loading metadata for box 'proycon/lamachine'
    default: URL: https://vagrantcloud.com/proycon/lamachine
The box you're attempting to add doesn't support the provider
you requested. Please find an alternate box or use an alternate
provider. Double-check your requested provider to verify you didn't
simply misspell it.

If you're adding a box from HashiCorp's Vagrant Cloud, make sure the box is
released.

Name: proycon/lamachine
Address: https://vagrantcloud.com/proycon/lamachine
Requested provider: [:hyperv]`

**Comment by proycon** [comment]
Okay, interesting. I looked into this and it seems you are running on Windows with Hyper-V enabled.  Hyper-V is a Microsoft virtualisation solution, but is not compatible with others.

> Enabling Hyper-V will cause VirtualBox, VMware, and any other virtualization technology to no longer work. See this blog post for an easy way to create a boot entry to boot Windows without Hyper-V enabled, if there will be times you will need other hypervisors.

Because Vagrant detects the Hyper-V it looks for a LaMachine solution with Hyper-V, but we only support virtualbox. The solution will be to disable Hyper-V and try again, vagrant refers to the following documentation to that end: http://www.hanselman.com/blog/SwitchEasilyBetweenVirtualBoxAndHyperVWithABCDEditBootEntryInWindows81.aspx 

I also found https://www.nextofwindows.com/how-to-enable-configure-and-use-hyper-v-on-windows-10

I may have to adapt the LaMachine bootstrap to automatically detect and warn about this, as I wasn't aware of it yet until now.
